lessons/module-3/unit-testing-react.md

```diff
@@ -212,52 +222,17 @@ const Grocery = ({ name, quantity, notes, purchased, starred, onPurchase, onStar
 export default Grocery;
 ```
 
-It's a functional, stateless component. Right now, we're not using a number of the 
```

Comment:
This section should not be removed. It's important to see how things have been done in the past, as many legacy projects that you all are likely to encounter will have these types of tests present.

Reply:
I revised in a way that may meet your interest in teaching the historical context along with the newer approach.